internal/pipeline/pipeline.go

```diff
@@ -228,6 +248,68 @@ func (f *Function) Validate() error {
 	return nil
 }
 
+// runner returns a fnRunner from the image and configs of
+// this function.
+func (f *Function) runner() (kio.Filter, error) {
```

Comment:
Split Function and its methods into a separate file. This file can then only include Pipeline

Reply:
done